### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by oliver-sanders**
Quick prettification of the readme file [Preview](https://github.com/oliver-sanders/cylc-flow/blob/readme/README.md):

* ~Rationalise~ Organise the badges.
* Add a Matrix badge for the "Cylc General" room.
* Bring project description in line with Cylc website.
* Outline Cylc 7 / Cylc 8 versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+![PyPI](https://img.shields.io/pypi/pyversions/cylc-flow.svg?color=green)
+[![PyPI](https://img.shields.io/pypi/v/cylc-flow.svg?color=yellow)](https://pypi.org/project/cylc-flow/)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/cylc-flow/badges/version.svg)](https://anaconda.org/conda-forge/cylc-flow)
+
+* Experimental alpha.
+* ZMQ (TCP) network layer.
+* In-browser GUI
+* On the `master` branch in the source code.
+
+The first official Cylc-8 release is not expected until late 2019.
```
Comment (kinow) [inline README.md:57]: Should we change to "until late 2020"? Or are we considering the alpha releases as official releases? Maybe we should say when the 8.0 is released in the readme?

**Review by kinow — APPROVED** [review]
Looks good to me! :+1:

File: README.md [hunk 2]
```diff
@@ -1,31 +1,64 @@
...
+* 7.8 - Python 2.6
+* 7.9 - Python 2.7
 
 [Quick Installation](INSTALL.md) |
 [Website](https://cylc.github.io/) |
-[Documentation](https://cylc.github.io/documentation) |
-[Contributing](CONTRIBUTING.md)
+[Documentation](https://cylc.github.io/documentation)
+
+### Cylc 8 (experimental)
```
Comment (kinow) [inline README.md:46]: @oliver-sanders just realized the Cylc 7 and Cylc 8 headers have different sizes? Was that intentional?

File: README.md (hunk 1 above)
Comment (oliver-sanders) [reply README.md:57]: Damn good point I forgot what year it is!

File: README.md (hunk 2 above)
Comment (oliver-sanders) [reply README.md:46]: Nope!

File: README.md [hunk 3]
```diff
@@ -1,31 +1,64 @@
...
+* HTTPS network layer.
+* PyGTK GUI.
+* On the `7.8.x` branch in the source code.
+* 7.8 - Python 2.6
+* 7.9 - Python 2.7
 
 [Quick Installation](INSTALL.md) |
 [Website](https://cylc.github.io/) |
-[Documentation](https://cylc.github.io/documentation) |
-[Contributing](CONTRIBUTING.md)
```
Comment (wxtim) [inline README.md:28]: I would not have thought that we'd want to lose this altogether.

Comment (oliver-sanders) [reply README.md:28]: Added new contributing section.

We should add quickstart, basic usage, etc once Cylc 8 is out.

File: README.md [hunk 4]
```diff
@@ -75,6 +77,25 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
...
+### Contributing
+
+[![Contributors](https://img.shields.io/github/contributors/cylc/cylc-flow.svg?color=9cf)](https://github.com/cylc/cylc-flow/graphs/contributors)
+[![Commit activity](https://img.shields.io/github/commit-activity/m/cylc/cylc-flow.svg?color=yellowgreen)](https://github.com/cylc/cylc-flow/commits/master)
+[![Last commit](https://img.shields.io/github/last-commit/cylc/cylc-flow.svg?color=ff69b4)](https://github.com/cylc/cylc-flow/commits/master)
+
+Contributions welcome:
+
+* Read the [contributing](CONTRIBUTING.md) page.
+* Development setup instations are in the
```
Comment (kinow) [inline README.md:89]: instations, instructions?

Comment (wxtim) [reply README.md:89]:
```suggestion
* Development setup instructions are in the
```

**Comment by oliver-sanders** [comment]
dev docs are still telling users to use `npm` so opened a fix - https://github.com/cylc/cylc-admin/pull/113

**Pull request merged**